### PR TITLE
feat(BUX-418, BUX-166): use Arc with gorillapool as a default option

### PR DESCRIPTION
**Pull request by wregulski**
# Pull Request Checklist

- [x] 📖 I created my PR using provided  : [CODE_STANDARDS](https://github.com/BuxOrg/bux/blob/main/.github/CODE_STANDARDS.md)
- [x] 📖 I have read the short Code of Conduct: [CODE_OF_CONDUCT](https://github.com/BuxOrg/bux/blob/main/.github/CODE_OF_CONDUCT.md)
- [x] 🏠 I tested my changes locally.
- [x] ✅ I have provided tests for my changes.
- [x] 📝 I have used conventional commits.
- [ ] 📗 I have updated any related documentation.
- [x] 💾 PR was issued based on the Github or Jira issue.

### CODE REVIEW

File: chainstate/transaction_info.go [hunk 1]
```diff
@@ -1,22 +1,28 @@
...
-	BlockHash     string          `json:"block_hash,omitempty"`    // mAPI, WOC
-	BlockHeight   int64           `json:"block_height"`            // mAPI, WOC
-	Confirmations int64           `json:"confirmations,omitempty"` // mAPI, WOC
-	ID            string          `json:"id"`                      // Transaction ID (Hex)
-	MinerID       string          `json:"miner_id,omitempty"`      // mAPI ONLY - miner_id found
-	Provider      string          `json:"provider,omitempty"`      // Provider is our internal source
-	MerkleProof   *bc.MerkleProof `json:"merkle_proof,omitempty"`  // mAPI 1.5 ONLY. Should be also supported by Arc in future
+	BlockHash     string             `json:"block_hash,omitempty"`    // mAPI, WOC
+	BlockHeight   int64              `json:"block_height"`            // mAPI, WOC
+	Confirmations int64              `json:"confirmations,omitempty"` // mAPI, WOC
```
Comment (pawellewandowski98) [inline chainstate/transaction_info.go:12]: I know that this has not changed in this PR but WOC should not be removed? we are not using it anymore

Comment (chris-4chain) [reply chainstate/transaction_info.go:12]: All are used. `BlockHash` `BlockHeight` in `sync_tx_service.go` and `Confirmations` by `checkRequirementMapi` function.

Comment (pawellewandowski98) [reply chainstate/transaction_info.go:12]: Sure, I was talking only about the comment `WOC`. Previously this data was probably gathered by WOC but we removed it some time ago or does it refer to something different?

Comment (chris-4chain) [reply chainstate/transaction_info.go:12]: I think You're right. @wregulski if you agree too - remove the `WOC` comment please :)

Comment (wregulski) [reply chainstate/transaction_info.go]: done

**Review by pawellewandowski98 — APPROVED** [review]

**Review by Nazarii-4chain — APPROVED** [review]

**Review by Nazarii-4chain — COMMENTED** [review]
Please check tests

**Pull request merged**